### PR TITLE
1_Updated docs

### PR DESCRIPTION
**Pull request by pdtran3k6**

### CODE REVIEW

File: tony-history-server/README.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 ```
 $ cd tony-history-server
 ```
 
+- Rename `conf/application.example.conf` to `conf/application.conf` and set 
+all the necessary configurations.
+
 - Build the project with Gradle (installed):
 ```
 $ gradle build
```
Comment (erwa) [inline tony-history-server/README.md]: Can we use `./gradlew`?

File: tony-history-server/README.md [hunk 2]
```diff
@@ -35,21 +38,29 @@ $ gradle testPlayBinary
...
 
-## Deploy to test cluster
-Before using the script, ensure that you have set execution permission (`chmod +x buildAndDeploy.sh`)
-
-- To bundle the history server app and deploy to another host
+## Production
+- Bundle the production binary:
 ```
-$ ./buildAndDeploy.sh user hostname.test.abc.com
+$ gradle dist
```
Comment (erwa) [inline tony-history-server/README.md]: Same

File: tony-history-server/conf/tony-site.sample.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <name>tony.https.keystore.algorithm</name>
+        <value>SunX509</value>
+    </property>
+    <property>
+        <name>tony.http.port</name>
+        <value>disabled</value>
+    </property>
+    <property>
+        <name>tony.secret.key</name>
+        <value>Prod</value>
```
Comment (erwa) [inline tony-history-server/conf/tony-site.sample.xml]: What does this `secret.key` do? The keystore password makes sense to me, but I'm not sure what this `secret.key` is for. Is the `secret.key` a password of sorts or is it just an identifier/label?

File: tony-history-server/README.md (hunk 1 above)
Comment (pdtran3k6) [reply tony-history-server/README.md]: Sure.

File: tony-history-server/conf/tony-site.sample.xml (hunk 3 above)
Comment (pdtran3k6) [reply tony-history-server/conf/tony-site.sample.xml]: https://www.playframework.com/documentation/2.6.x/ApplicationSecret.

File: tony-history-server/README.md (hunk 2 above)
Comment (pdtran3k6) [reply tony-history-server/README.md]: Sure.

File: tony-history-server/conf/tony-site.sample.xml (hunk 3 above)
Comment (erwa) [reply tony-history-server/conf/tony-site.sample.xml]: Thanks. That doc says "Anyone that can get access to the secret will be able to generate any session they please, effectively allowing them to log in to your system as any user they please."

So we should make sure the production secret we use is not checked in to source control and not in the tony-site.xml on the client side.

Comment (erwa) [reply tony-history-server/conf/tony-site.sample.xml]: I would suggest changing the value of `tony.secret.key` in the example to `changeme` (which is the default for `play.http.secret.key`.

**Pull request merged**